tasks.py

```diff
@@ -0,0 +1,857 @@
+            warn=True,
+            hide=not verbose,
+        )
+        run("sudo chmod +x /usr/bin/rabbitmqadmin ", warn=True, hide=not verbose)
+        run(
+            "sudo chown -R rabbitmq:rabbitmq /var/lib/rabbitmq/",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo rabbitmqadmin declare queue  --vhost=/ name=email",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo rabbitmqadmin declare queue  --vhost=/ name=builds",
+            warn=True,
+            hide=not verbose,
+        )
+        if run("rabbitmqadmin list queues", warn=True, hide=not verbose).ok:
+            run('echo "===>rabbitmq {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "===>rabbitmq {}"'.format(MESSAGE_FAILED))
+
+    elif PLATFORM == "Darwin":
+        if run("command -v rabbitmqctl", hide=not verbose, warn=True):
+            run("echo '===>rabbitMq {}'".format(MESSAGE_ALREADY_INSTALLED))
+        else:
+            if run("brew install rabbitmq").ok:
+                run('echo "===>rabbitmq {}"'.format(MESSAGE_OK))
+            else:
+                run('echo "===>rabbitmq {}"'.format(MESSAGE_FAILED))
+        run("sudo rabbitmqctl add_user lookit-admin admin", warn=True, hide=not verbose)
+        run(
+            "sudo rabbitmqctl set_user_tags lookit-admin administrator",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo rabbitmqctl set_permissions -p / lookit-admin '.*' '.*' '.*'",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo brew services stop rabbitmq-server.service",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo brew services start rabbitmq-server.service",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo brew services enable rabbitmq-server.service",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo rabbitmq-plugins enable rabbitmq_management",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo chown -R rabbitmq:rabbitmq /var/lib/rabbitmq/",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo rabbitmqadmin declare queue  --vhost=/ name=email",
+            warn=True,
+            hide=not verbose,
+        )
+        run(
+            "sudo rabbitmqadmin declare queue  --vhost=/ name=builds",
+            warn=True,
+            hide=not verbose,
+        )
+        run("sudo rabbitmqadmin list queues", warn=True, hide=not verbose)
+    else:
+        run("echo {}".format(MESSAGE_WRONG_PLATFORM))
+
+    return None
+
+
+@task
+def postgresql(c, verbose=False):
+    """ postgresql invoke task.
```

Comment:
Same comment as before. It's nitpicky but it keeps things looking clean and professional